build.assets/makefiles/base/agent/planet-agent.service

```diff
@@ -14,7 +14,7 @@ LimitNOFILE=40000
 LimitNPROC=1048576
 ExecStartPre=/bin/systemctl is-active etcd.service
 ExecStartPre=/bin/systemctl is-active serf.service
-ExecStart=/usr/bin/planet agent \
+ExecStart=/usr/bin/planet agent --debug \
```

Comment:
this will just spam the logs won't it? Shouldn't this instead set the level to info and focus on more informative logs at info?